### PR TITLE
Set --quiet option on builds to suppress some output

### PR DESCRIPTION
**Pull request by markelliot**

### CODE REVIEW

File: src/main/groovy/com/palantir/gradle/docker/PalantirDockerPlugin.groovy [hunk 1]
```diff
@@ -91,7 +91,7 @@ class PalantirDockerPlugin implements Plugin<Project> {
                 into dockerDir
             }
 
-            List<String> buildCommandLine = ['docker', 'build']
+            List<String> buildCommandLine = ['docker', 'build', '--quiet']
```
Comment (markelliot) [inline src/main/groovy/com/palantir/gradle/docker/PalantirDockerPlugin.groovy]: @uschi2000 curious for your thoughts here, but builds for some of our projects are generating huge reports -- think we can eliminate a lot of junk build content by setting --quiet here.

Comment (uschi2000) [reply src/main/groovy/com/palantir/gradle/docker/PalantirDockerPlugin.groovy]: Can we inspect whether Gradle was called with --info or --debug and set this accordingly?

**Comment by markelliot** [comment]
That seems pretty reasonable.

**Comment by markelliot** [comment]
@uschi2000, how's this?

**Comment by uschi2000** [comment]
👍

**Pull request merged**